RecoLocalTracker/SiStripRecHitConverter/plugins/StripCPEESProducer.cc

```diff
@@ -12,6 +19,25 @@
 #include "CondFormats/SiStripObjects/interface/SiStripBadStrip.h"
 #include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
The following three headers can be removed now:
```C++
#include "CondFormats/SiStripObjects/interface/SiStripNoises.h"
#include "CondFormats/SiStripObjects/interface/SiStripApvGain.h"
#include "CondFormats/SiStripObjects/interface/SiStripBadStrip.h"
```

And maybe you can profit and move the FWCore one on top, together with the other include's of FWCore headers

Reply:
Done